Intersect.Server/Admin/Actions/ActionProcessing.cs

```diff
@@ -447,5 +447,24 @@ public static void ProcessAction(Player player, WarpToMeAction action)
                 );
             }
         }
+
+        //ReturnToOverworld
+        public static void ProcessAction(Player player, ReturnToOverworldAction action)
+        {
+            var target = Player.FindOnline(action.PlayerName);
+            if (target != null)
```

Comment:
Invert the nested ifs and just return early, and don't one-liner the if (...) return;

Reply:
Done